Competition2018/src/main/java/competition/subsystems/drive/commands/RotateToHeadingCommand.java

```diff
@@ -0,0 +1,51 @@
+package competition.subsystems.drive.commands;
+
+import xbot.common.command.BaseCommand;
+import xbot.common.injection.wpi_factories.CommonLibFactory;
+import xbot.common.math.PIDFactory;
+import xbot.common.math.PIDManager;
+import xbot.common.subsystems.drive.control_logic.HeadingModule;
+import com.google.inject.Inject;
+import competition.subsystems.drive.DriveSubsystem;
+
+public class RotateToHeadingCommand extends BaseCommand{
+
+    double goal;
+    double power;
+    double heading;
+    final CommonLibFactory clf;
```

Comment:
You don't actually need this variable, since you only use it in the constructor.

Reply:
removed variable "heading"